stance/motion/Motion.py

```diff
@@ -0,0 +1,61 @@
+from abc import ABC, abstractmethod
+
+
+class Motion(ABC):
+    @abstractmethod
+    def create_template_skeletons(self, front_view_points, profile_view_points):
+        """
+        Using the body points from a user standing from two views, create
+        template skeletons for benchmark zones that define the specific motion.
+
+        Parameters
+        ----------
+        front_view_points : array of points of the front view of the user
```

Comment:
Here "array of points of ..." should be a proper type. If we need a container type to hold the COCO points, we should make one, probably a `namedtuple`